### PR TITLE
feat: improve series configuration style and layout

### PR DESCRIPTION
**Pull request by joaoviana**
Relates to: [#11939](https://github.com/lightdash/lightdash/issues/11939) 

### Description:

This PR: 
- gets the series colours correctly after merge of #11611 
- creates a series tab
- allows the user customising the series through a popover
- adds small improvements to the design
- changes the `styling` tab to `display` 

> [!IMPORTANT]
> This PR does not cover the grouped series generation. It will be a separate PR because it requires type changes.

### Reviewer actions

- [ ] I have manually tested the changes in the preview environment
- [ ] I have reviewed the code
- [ ] I understand that "request changes" will block this PR from merging

### CODE REVIEW

**Comment by joaoviana** [comment]
getting a review first from @PriPatel once Render env is up

File: packages/common/src/visualizations/CartesianChartDataModel.ts [hunk 1]
```diff
@@ -22,6 +22,19 @@ import {
 } from './types';
 import { type IResultsRunner } from './types/IResultsRunner';
 
+export const getDefaultColor = (index: number, orgColors?: string[]) => {
```
Comment (joaoviana) [inline packages/common/src/visualizations/CartesianChartDataModel.ts:25]: this was a static method that can actually be used across other files.

File: packages/common/src/visualizations/CartesianChartDataModel.ts [hunk 2]
```diff
@@ -497,12 +501,18 @@ export class CartesianChartDataModel {
                     (s) => s.yAxisIndex === index,
                 )?.color;
 
+                const seriesType = Object.values(display?.series || {}).find(
```
Comment (joaoviana) [inline packages/common/src/visualizations/CartesianChartDataModel.ts:504]: this was reverted/removed from #11611

**Comment by PriPatel** [comment]
https://www.notion.so/lightdash/Review-2-129a63207a7a80a88fb0d1f54e2a7eaa

**Comment by joaoviana** [comment]
Closing as it's been deprioritised

**Pull request closed without merging**